### PR TITLE
[Documentation] Created Survey Accounts module help text

### PR DESCRIPTION
**Pull request by knoll-alex**
## Brief summary of changes
I created a new help text for the Survey Accounts module, since it didn't have one. I integrated content from @leighevans in 2017 which should have been committed at that time.

#### Testing instructions (if applicable)

1. Ensure all information is correct and easy to read
2. Ensure help text shows up in the LORIS front-end, with correct formatting.

#### Links to related tickets (GitHub, Redmine, ...)

https://github.com/aces/Loris/issues/5241
Original PR was never completed: https://github.com/aces/Loris/pull/3390

### CODE REVIEW

File: modules/survey_accounts/help/survey_accounts.md [hunk 1]
```diff
@@ -1,43 +1,20 @@
...
+This module is used to create and manage survey forms that are sent to participants in a study.
 
-The survey accounts module is intended to manage the surveys which
-are sent to participants in a study. It is used to generate a survey
-key, and (optionally) automatically send it to the participant, if
-their email address is provided.
+Each survey form is generated with a unique URL. This survey link URL can be emailed directly to respondents for survey completion from home, or can be used in the clinic to capture data directly from participants, e.g. on a tablet in the waiting room during a study visit. 
 
-## Intended Users
+Use the Selection Filters to search for existing surveys by PSCID, Visit Label, Instrument, or the email of a participant who has already been sent a survey form. You can use any selection of filters needed. Click **Clear Filters** to clear the current selection of filters. Click **Show Data** to display the results in a data table.
```
Comment (christinerogers) [inline modules/survey_accounts/help/survey_accounts.md]: 2 quick comments on this para: 
* `You can use any selection of filters needed.` - can we remove? sounds a little clunky
* `Click **Show Data**` : There is no more Show Data button (This module was recently Reactified)

File: modules/survey_accounts/help/survey_accounts.md [hunk 2]
```diff
@@ -1,43 +1,20 @@
...
+* *Sent*: Survey was created, and an email with the unique survey URL was sent to the survey respondent. This is the default status once a survey is created and sent by email within LORIS.<br>
+* *In Progress*: Survey was opened and accessed by anyone with the link, including study personnel (not just the intended respondent).<br>
+* *Complete*: Survey was completed and submitted. After this stage, the respondent will not be able to go back and change any entries. No data will be visible via the URL once a survey is completed: clicking on the URL will display a page with the message *"Data has already been submitted"*.
 
-## Permissions
+You can create a new survey manually with data that hasn’t yet been entered into LORIS. Click **Add Survey**, which takes you to a new page. Enter the required information and click **Create Survey** to simply generate a URL (which you can send to the candidate), or **Email Survey** to automatically send the URL to the candidate. If you choose Email Survey, you’ll be able to customize the content of the email in a pop-up window (which will contain an editable generic message).
```
Comment (christinerogers) [inline modules/survey_accounts/help/survey_accounts.md]: `You can create a new survey manually with data that hasn't yet been ` 
This phrase sounds a little ambiguous to me re `create` (a new instrument?) and `data` (like participant's email, which is considered metadata but not survey data). 

I'm not 100% sure what might sound clearer here, let's come back to this together.

File: modules/survey_accounts/help/survey_accounts.md (hunk 1 above)
Comment (knoll-alex) [reply modules/survey_accounts/help/survey_accounts.md]: Thanks! Sorry I forgot about the Show Data - removed both of those lines as requested.

**Comment by knoll-alex** [comment]
Screenshot 2019-10-02 09 17 30

Here's a screenshot of how it renders - unfortunately the bullet points created for Status aren't rendering properly, along with italics in that section. Does all of markdown work within LORIS?

**Comment by driusan** [comment]
This has conflicts 

```
Conflicting files
modules/survey_accounts/help/survey_accounts.md
```

**Comment by johnsaigle** [comment]
@knoll-alex 
> Does all of markdown work within LORIS?

No, at this time only a subset of Markdown works within LORIS:

https://github.com/aces/Loris/blob/3d6d2a060e349b34093146134e13caff440747e3/jsx/Markdown.js#L7-L12

So it looks like we don't support formatted bullet points at this time.

I'll create a new issue proposing that we add bullet points. I think it would be useful.

File: modules/survey_accounts/help/survey_accounts.md (hunk 2 above)
Comment (knoll-alex) [reply modules/survey_accounts/help/survey_accounts.md]: Revisiting - I must be misunderstanding this function. Making a note here so we can chat about it.

**Comment by christinerogers** [comment]
Hi @knoll-alex any updates on this ?

**Comment by knoll-alex** [comment]
> Hi @knoll-alex any updates on this ?

Here's a screenshot of the latest commit (I updated formatting to bold defined terms).

There's one piece of info in Permissions that we need to connect on - see my comment from a few days ago

Screenshot 2019-11-06 17 26 25

File: modules/survey_accounts/help/survey_accounts.md (hunk 2 above)
Comment (christinerogers) [reply modules/survey_accounts/help/survey_accounts.md]: looks fine - i just re-reviewed and can't think of a better wording...

**Comment by christinerogers** [comment]
This looks great, and the rebase was 💯 👍 

Could you post a final screenshot and we'll approve for merging?  
Feel free to edit out prior screenshots (so they don't clutter the comments) - you can edit the comment containing it to not display the image

**Comment by knoll-alex** [comment]
Screenshot 2019-11-13 11 47 48

@christinerogers 

FYI I may revisit this one as I work through [PR #5576](https://github.com/aces/Loris/issues/5576) - I think it can be paired down more but let's still close this PR.

**Pull request merged**